IBAnimatable/IBEnum.swift

```diff
@@ -5,17 +5,19 @@
 
 import Foundation
 
+
+
 /**
  A protocol provides extension method for converting `String` into `enum`.
  Because `@IBInspectable` property can not support `enum` directly. To provide both `enum` API in code and `@IBInspectable` supported type `String` in Interface Builder, we use `IBEnum` to bridge Swift `enum` and `String`
  */
-public protocol IBEnum: StringLiteralConvertible {
+public protocol IBEnum {
   /**
    Initializes a swift `enum` with provided optional string
    
    - Parameter string: The optional string to be converted into `enum`.
    */
-  init(string: String?)
+  init?(string: String?)
```

Comment:
Hum, 
Me too, I am not very happy with this. 
But the problematic is : 
Some Enums (For the moment, `UIBlurEffectStyle`) need to have a failable `init` since there is no `none` case.
But other Enums Like `Masktype` and `AnimationType` don't need failable `init`. 
I first attempted to create a second protocol (IBOptionalEnum) which require a failabile `init`.
But It seems that a class that provides a non-bailable init still conforms to a protocol requiring a failable init.
That's why I decided to let it like that.
What do you think to be the best solution ?